app/models/sms/adapters/adapter.rb

```diff
@@ -1,11 +1,11 @@
 # models a generic sms adapter. should be subclassed.
-require 'net/http'
+require "net/http"
 class Sms::Adapters::Adapter
 
   attr_writer :deliveries
 
   # checks if this adapter recognizes an incoming http receive request
-  def self.recognize_receive_request?(request)
+  def self.recognize_receive_request?(_request)
```

Comment:
Yeah that's what I thought. I think let's leave them out for new and we can consider making that part of our style guide.